Tasks/CopyFilesV2/copyfiles.ts

```diff
@@ -16,7 +16,11 @@ let flattenFolders: boolean = tl.getBoolInput('flattenFolders', false);
 // determine the relative path of each found file (substring using sourceFolder.length).
 sourceFolder = path.normalize(sourceFolder);
 
-let allPaths: string[] = tl.find(sourceFolder); // default find options (follow sym links)
+// Default find options except for allowing broken sym links.
```

Comment:
Does this change behavior or keep it the same? Do we have tests to cover this?

Reply:
The only difference in behavior is that it allows broken symlinks (doesn't throw when it encounters one). We don't have coverage here, we do in the task-lib itself though. IMO it doesn't feel like this task's responsibility to test that specific functionality and testing broken symlinks is kinda gross so I think its fine, thoughts?